lib/room.h

```diff
@@ -302,6 +302,19 @@ namespace QMatrixClient
 
             MemberSorter memberSorter() const;
 
+            bool processCall(Room* room, const RoomEvent* event);
```

Comment:
How much should this be in the public interface? Maybe move it to `Room::Private`?